pkg/reconciler/accessor/errors_test.go

```diff
@@ -34,17 +35,14 @@ func TestIsNotOwned(t *testing.T) {
 		want: true,
 	}, {
 		name: "other error",
-		err: Error{
```

Comment:
Not sure if we should still use the `Error` to maintain consistency?

Reply:
Why not `NewAccessorError("test error", "SomeOtherReason")`?

Reply:
@taragu and @dgerd 
This scenario to hit this part of [code](https://github.com/knative/serving/blob/master/pkg/reconciler/accessor/errors.go#L46)
So it error should not be of type `Error` or `NewAccessorError("test error", "SomeOtherReason") (because NewAccessorError returns Error again)`

Reply:
Ah I see. Thanks for the clarification!